isaac/config/behaviors/inspection.config

```diff
@@ -305,6 +262,51 @@ parameters = {
     max = 20,
     unit = "integer",
     description = "Number of times the inspection will retry to reach a pose"
+  },{
+    id = "pan_min",
+    reconfigurable = true,
+    type = "double",
+    default = -180,
+    min = -180,
+    max = 0,
+    unit = "degrees",
+    description = "Number of times the inspection will retry to reach a pose"
```

Comment:
Several of these new parameters have an accidentally copy/pasted description field. Descriptions could be pulled from the doc strings in the reference implementation.